plugins/opensource/pagebreak.md

```diff
@@ -14,10 +14,6 @@ This plugin adds page break support and enables a user to insert page breaks in
 
 It also adds a toolbar button and a menu item `Page break` under the `Insert` menu dropdown.
 
-**Type:** `String`
-
-**Default Value:** `"<!-- pagebreak -->"`
```

Comment:
This probably needs to be moved to the `pagebreak_separator` setting instead of just being deleted.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1838/commits/d611ea2b482f79339c7f50769540e80a8936e516